conference/models.py

```diff
@@ -336,9 +343,15 @@ class AttendeeProfile(models.Model):
     phone = models.CharField(
         _('Phone'),
         max_length=30, blank=True,
-        help_text=_('Enter a phone number where we can contact you in case of administrative issues.<br />Use the international format, eg: +39-055-123456'),
+        help_text=_(
+            'Enter a phone number where we can contact you in case of '
+            'administrative issues.<br>Use the international format, '
+            'eg: +39-055-123456'
+        ),
+    )
+    gender = models.CharField(
+        max_length=1, choices=ATTENDEEPROFILE_GENDER, blank=True, default=""
```

Comment:
Regarding the default, I'm wondering whether it's worth it to distinguish between users who have yet to provide a value and users who selected `prefer not to say`. WDYT?

Reply:
I made it so we have also blank as default.